### PR TITLE
Add "organization" field to more events

### LINKED ISSUE #2950: Not all Webhook events that provide organizations, are available in the library.

### PR DESCRIPTION
**Pull request by billnapier**
This should cover all events that GitHub says have an organization field.

Fixes #2950

### CODE REVIEW

File: github/event_types.go
Comment (gmlewis) [inline github/event_types.go:206]: Let's please name them all `Org`.

Comment (billnapier) [reply github/event_types.go]: I left the existing ones, as I wasn't sure if you wanted a breaking change here or not.  Happy to go change them all if you don't mind the breakage.

Comment (gmlewis) [reply github/event_types.go]: Oh, sorry! That will teach me to do code reviews on my Android phone. 😂

Hmmm... consistency is nice as long as we document our changes.

Sure, let's be consistent and I'll mark this as a breaking API change. 😁

Comment (gmlewis) [reply github/event_types.go]: Hmmm... I'm no longer on my phone, and I'm not understanding this statement:

> I left the existing ones, as I wasn't sure if you wanted a breaking change here or not. Happy to go change them all if you don't mind the breakage.

I dont see any existing `organization` fields in any of the structs you modified, so I don't think this is a breaking API change afterall, correct?

**Review by gmlewis — APPROVED** [review]
Thank you, @billnapier !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by WillAbides — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @WillAbides !
Merging.

**Pull request merged**
